app/controllers/events_controller.rb

```diff
@@ -22,15 +22,15 @@ def show
   end
 
   def show_category
-    @type = GetIntoTeachingApiClient::TypesApi.new.get_teaching_event_types.find do |type|
+    @type = GetIntoTeachingApiClient::PickListItemsApi.new.get_teaching_event_types.find do |type|
       I18n.t("event_types.#{type.id}.name.plural").parameterize == params[:category]
     end
 
     render(template: "errors/not_found", status: :not_found) && return if @type.nil?
 
     @event_search = Events::Search.new(event_filter_params.merge(type: @type.id))
     all_results = @event_search.query_events(MAXIMUM_EVENTS_IN_CATEGORY)
-    @events = paginate(all_results[@type.id.to_sym])
+    @events = paginate(all_results[@type.id.to_s.to_sym])
```

Comment:
Is the `.to_s` necessary here?

Reply:
Yeah annoyingly you can't do `int.to_sym` you have to go to string first 🤷